sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerWithSparkContextSuite.scala

```diff
@@ -102,14 +102,14 @@ trait ThriftServerWithSparkContextSuite extends SharedThriftServer {
 
       spark.sparkContext.addSparkListener(listener)
       try {
-        statement.execute(s"SET ${SQLConf.THRIFTSERVER_QUERY_TIMEOUT.key}=1")
+        statement.execute(s"SET ${SQLConf.THRIFTSERVER_QUERY_TIMEOUT.key}=500")
```

Comment:
Wait, is it 500 seconds?

Reply:
Can you elabourate a bit more? It sleeps 3 seconds below via `Thread.sleep`.

Reply:
Sorry I misread by the error msg.

The timeout error msg is confused since the timeout config `spark.sql.thriftServer.queryTimeout` can't change the `Statement.queryTimeout`, then the error msg is always `Query timed out after 0 seconds`.

Back to the flaky test, seems the set config sql will timeout sometime if we limit the queryTime to 1 second.